lib/chutney/TorNet.py

```diff
@@ -764,6 +764,19 @@ def _genRouterKey(self):
             sys.exit(1)
         self._env['fingerprint'] = fingerprint
 
+
+    def _setEd25519Id(self):
+        """Make chutney check for relay microdescriptors before verifying"""
+        datadir = self._env['dir']
+        key_directory =  os.path.join(datadir, 'keys', "ed25519_master_id_public_key")
+        with open(key_directory ,'rb') as f:
+            f.seek(32)
```

Comment:
What happens if the file is empty?

Let's check the return value of seek(). The seek documentation is here:
https://docs.python.org/3.5/library/io.html#io.IOBase.seek

I'll leave you to write the code that checks the return value.

Let's also use a named constant, rather than just saying "32":
```suggestion
            ED25519_KEY_POSITION = 32
            f.seek(ED25519_KEY_POSITION)
```